streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestUtil.java

```diff
@@ -128,4 +128,22 @@ public static void sleep(final long duration) {
         } catch (final Exception ignore) { }
     }
 
+    public static class VerificationResult {
```

Comment:
Should we remove `VerificationResult` from SmokeTestDriver in this PR, too? Or would it cause too many updates we don't want?